### ISSUE TITLE
Inconsistent use of border/border_color and borderwidth/border_width

### ISSUE THREAD

**Issue by m-col** [qtile/qtile#2825]
# Issue description

Title - some of the widgets are not consistent with other configurables with these config options. See [this comment](https://github.com/qtile/qtile/pull/2675#issuecomment-927275743) for more details.

It would be nice to make these consistent with other configurables, perhaps adding a migration and deprecation notice for a version or two.

# Qtile version

0.18.1

**Comment by elParaguayo** [comment]
Agreed - this would be good. Would need a good `migrate` script too.

Perhaps we could also have some docs listing common parameter names.

**Comment by luisalcarasr** [comment]
Same case for `single_border_width`/`border_on_single` and `single_margin` and `margin_on_single`.

**Comment by elParaguayo** [comment]
Agreed. I have a slight preference for the `single_` names.
